src/hooks/motionWidgets/stakingWidget/index.ts

```diff
@@ -0,0 +1,7 @@
+export { default as useStakingControls } from './useStakingControls';
```

Comment:
I like this pattern of wrapping all logic into a single hook that exposes only what is consumed by the view its named after. It keeps the view neat.

The only thing I would like to change (per my comment below) is to colocated the hook with its component in the file system, and save the top level "hooks" folder for hooks that get reused around the app. 

👍 or 👎?

Reply:
Sure, we need to refactor hooks anyway (at least their structure), since they've grown exponentially, and it's getting hard to work with them

Just maybe don't do it in this PR, since it's massive as it is already